glide.yaml

```diff
@@ -108,6 +108,8 @@ import:
   version: 1eb03e3cc9d345307a45ec82bd3016cde4bd4464
 - package: github.com/cznic/y
   version: 9fdf92d4aac058959f814606bb729ed50f5e4240
+- package: github.com/dustin/go-humanize
```

Comment:
I don't know either. We didn't set version for it and glide will update this automaticly.